src/plugins/data-card/data-card-tile.tsx

```diff
@@ -53,8 +54,23 @@ export const DataCardToolComponent: React.FC<ITileProps> = observer((props) => {
         return model.title;
       }
     });
+    adjustHeight();
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+  const adjustHeight = useCallback(() => {
+    if (!tileElt) return;
+    if (readOnly) onRequestRowHeight(model.id, height);
+    if (!readOnly){
+      const uiHeight = tileElt?.querySelector(".data-card-container")?.scrollHeight || 0;
+      const heightDiff = height ? height - uiHeight : 0;
+      heightDiff < 30 && onRequestRowHeight(model.id, uiHeight + 60);
```

Comment:
I'd usually think of a `diff` as change from the old to the new.  So if the tile never shrinks, then I'd think the heightDiff should always be positive. So I'd recommend changing this to:
```suggestion
      const heightDiff = height ? uiHeight - height : 0;
      heightDiff > 30 && onRequestRowHeight(model.id, uiHeight + 60);
```

Reply:
I ended up keeping the calculation intact but changing the names of the variables (and adding a constant).  I did try a version with your calculation but for some reason it was not working, let me know if it makes more sense now or if you still think it should be inverted.